nsqlookupd/options.go

```diff
@@ -16,7 +18,7 @@ type Options struct {
 	InactiveProducerTimeout time.Duration `flag:"inactive-producer-timeout"`
 	TombstoneLifetime       time.Duration `flag:"tombstone-lifetime"`
 
-	Logger logger
+	Logger app.Logger
```

Comment:
`nsqlookupd` is technically a public package and `app` is now an `internal` one - seems like that might be undesirable?